mailer/tests.py

```diff
@@ -595,30 +599,45 @@ def test_message_log(self):
 
     def test_message_str(self):
         with self.settings(MAILER_EMAIL_BACKEND="django.core.mail.backends.locmem.EmailBackend"):
-            mailer.send_mail("Subject Msg", "Body", "msg1@example.com", ["rec1@example.com"])
+            mailer.send_mail("Subject Msg 中", "Body 中", "msg1@example.com", ["rec1@example.com"])
 
             self.assertEqual(Message.objects.count(), 1)
 
             msg = Message.objects.get()
-            self.assertEqual(
-                str(msg),
-                'On {0}, \"Subject Msg\" to rec1@example.com'.format(msg.when_added),
-            )
+            # Don't need to encode to utf-8 on Python 3.
+            if six.PY2:
```

Comment:
Using `six.text_type(msg)` doesn't work. I don't like the conditional either. I think I'll just write an integration test that loads the admin page to demonstrate the issue. I think that's easier for everybody to understand.